website/docs/reference/faq.md

```diff
@@ -97,3 +97,30 @@ Link to support page
     Keep in mind that the ports will increment as you connect multiple devices.
   </div>
 </details>
+
+<details>
+  <summary>How do I update the dashboard firmware?</summary>
+  <div>
+     All latest updates are available online with detailed instructions that will take you step by step to a successful update.
+    <br/><br/>
+    - You can find more details at: <a href="https://amiga.farm-ng.com/docs/dashboard/dashboard-fw#wired-updates">Dashboard Firmware Updates</a>.
+    </div>
+</details>
+
+<details>
+  <summary>How can I create a new app for the brain?</summary>
+  <div>
+     Custom applications can be created on the brain from anywhere. We have a detailed tutorial on creating your first custom app on your Amiga.
+    <br/><br/>
+    - You can find more details at: <a href="https://amiga.farm-ng.com/docs/brain/brain-apps">Developing Custom Applications</a>.
+    </div>
+</details>
+
+<details>
```

Comment:
Since you provide details of where to go for different types of errors on the support page, you can either add another drop down linking to the support page or just link to that page in this drop down instead of directly to the issues page

Reply:
There's also a place holder for 'where do i get support' at the top of this page where this should be moved and you should delete that coming soon note

Reply:
I will get rid of the place holder and just keep the direct link to the issues page.